### PR TITLE
user-guide: add some notes on operators sections

### PR DESCRIPTION
**Pull request by danielerez**

### CODE REVIEW

File: docs/user-guide.md [hunk 1]
```diff
@@ -276,11 +278,13 @@ metadata:
 spec:
   installPlanApproval: Automatic
   name: elasticsearch-operator
-  source: redhat-operator-index
+  source: cs-redhat-operator-index
```
Comment (nmagnezi) [inline docs/user-guide.md:281]: why is this prefix being added?

Comment (danielerez) [reply docs/user-guide.md]: It's actually a different issue. Moved to another PR: https://github.com/openshift/appliance/pull/160

**Review by nmagnezi — APPROVED** [review]
/lgtm

**Pull request merged**